### PR TITLE
Add Arabic localization

### PR DESCRIPTION
**Pull request by YazeedAlKhalaf**
This PR adds the Arabic language localization.

Reviewers:
- @pr-mais

### CODE REVIEW

File: Sources/KeyboardShortcuts/Localization/ar.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"record_shortcut" = "سجل اختصار";
```
Comment (pr-Mais) [inline Sources/KeyboardShortcuts/Localization/ar.lproj/Localizable.strings:1]:
```suggestion
"record_shortcut" = "سجل اختصاراً";
```

Not familiar with the context here but this string is used when we want to record a new keyboard shortcut, right?

File: Sources/KeyboardShortcuts/Localization/ar.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+"press_shortcut" = "اضغط الاختصار";
```
Comment (pr-Mais) [inline Sources/KeyboardShortcuts/Localization/ar.lproj/Localizable.strings:2]:
```suggestion
"press_shortcut" = "اضغط على الاختصار";
```

File: Sources/KeyboardShortcuts/Localization/ar.lproj/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+"keyboard_shortcut_used_by_menu_item" = "لا يمكن استخدام اختصار لوحة المفاتيح هذا لأنه مستخدم بواسطة عنصر القائمة "٪ @".";
+"keyboard_shortcut_used_by_system" = "لا يمكن استخدام اختصار لوحة المفاتيح هذا لأنه مستخدم على مستوى النظام.";
```
Comment (pr-Mais) [inline Sources/KeyboardShortcuts/Localization/ar.lproj/Localizable.strings:4]:
```suggestion
"keyboard_shortcut_used_by_system" = "لا يمكن استخدام اختصار لوحة المفاتيح هذا لأنه مستخدم مسبقاً على مستوى النظام.";
```

File: Sources/KeyboardShortcuts/Localization/ar.lproj/Localizable.strings [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+"keyboard_shortcuts_can_be_changed" = "يمكن تغيير معظم اختصارات لوحة المفاتيح على مستوى النظام في "تفضيلات النظام"> لوحة المفاتيح> الاختصارات ".";
```
Comment (pr-Mais) [inline Sources/KeyboardShortcuts/Localization/ar.lproj/Localizable.strings:5]:
```suggestion
"keyboard_shortcuts_can_be_changed" = "يمكن تغيير معظم اختصارات لوحة المفاتيح على مستوى النظام في "تفضيلات النظام > لوحة المفاتيح > الاختصارات ".";
```

**Comment by YazeedAlKhalaf** [comment]
Thanks @pr-Mais 🚀

**Pull request merged**